components/input/ResizableTextArea.tsx

```diff
@@ -48,6 +48,17 @@ class ResizableTextArea extends React.Component<TextAreaProps, TextAreaState> {
     }
   }
 
+  handleResize = (size: { width: number; height: number }) => {
+    const { autoSize, autosize, onResize } = this.props;
```

Comment:
It seems like you're just moving the logic concerning `autoSize` and `autosize`, but while you're at it, it may be worth considering whether we can use more informative naming. A difference in case is not very informative, especially to new contributors, including myself.

Reply:
Thanks for your code review, What's your suggestion for this.